provisioner/roles/splunk_enterprise/files/lets-encrypt-x3-cross-signed.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
What is this about?

Reply:
It's needed for the provisioning of a letsencrypt cert, I borrowed it from `./provisioner/roles/aws_dns/files/lets-encrypt-x3-cross-signed.key` along with logic from that role.

Reply:
Ok.